internal/turso/databases.go

```diff
@@ -26,8 +27,21 @@ type Database struct {
 
 type DatabasesClient client
 
-func (d *DatabasesClient) List() ([]Database, error) {
-	r, err := d.client.Get(d.URL(""), nil)
+func (d *DatabasesClient) List(options map[string]string) ([]Database, error) {
```

Comment:
When possible, default to typed options. Using a struct here that has `Group` and `Schema` fields could be a good option. You could also add an encode method to it to encapsulate the logic.

Reply:
Thanks! Will PR the changes